### PR TITLE
Add version: UniversityOfWaikato.Weka version 3.8.6

### PR DESCRIPTION
**Pull request by jontyman**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/119129)

### CODE REVIEW

**Comment by jontyman** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Standard error:  Exception during executable launch operation System.AggregateException: One or more errors occurred. (The specified executable is not a valid application for this OS platform.)  ---
> System.ComponentModel.Win32Exception (216): The specified executable is not a valid application for this OS platform. 
--- End of stack trace from previous location where exception was thrown ---    --- End of inner exception stack trace ---    at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)

File: manifests/u/UniversityOfWaikato/Weka/3.8.6/UniversityOfWaikato.Weka.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: UniversityOfWaikato.Weka
+PackageVersion: 3.8.6
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/u/UniversityOfWaikato/Weka/3.8.6/UniversityOfWaikato.Weka.installer.yaml:9]:
```suggestion
- Architecture: x64
```
`x86` validations are performed on a 32-bit OS, so the error above might indicate that this is a 64-bit program.

Comment (jontyman) [reply manifests/u/UniversityOfWaikato/Weka/3.8.6/UniversityOfWaikato.Weka.installer.yaml:9]: Thanks for the suggestion @stephengillie . It does install to "C:\Program Files\Weka-3-8-6"

**Review by stephengillie — APPROVED** [review]

**Pull request merged**